src/Service/BoardHelper.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Service;
+
+use App\Entity\Board;
+
+class BoardHelper
+{
+    public function getStatusesByBoard(Board $board): array
+    {
+        $rawStatuses = explode(
+            PHP_EOL,
+            $board->getStatuses()
+        );
+
+        $trimmedStatuses = [];
+        foreach ($rawStatuses as $rawStatus) {
+            $trimmedStatuses[$rawStatus] = trim($rawStatus);
+        }
+
+        return $trimmedStatuses;
+    }
```

Comment:
This function seems to assume that the result will be used as options, but `getStatusesByBoard` should probably just return a list of statuses (that can be converted to options when needed as such).

You can use

```php
preg_split('/'.preg_quote(PHP_EOL, '/').'/', $board->getStatuses(), -1, PREG_SPLIT_NO_EMPTY)
```
to split and trim in one go.

Alternatively, you can use

```php
array_filter(array_map('trim', explode(PHP_EOL, $board->getStatuses()))
```

Reply:
An even better approach will probably be to store the list of statuses as a list of strings in the database. 

Using a data transformer you can convert a list of strings to a newline separated text when editing in the form and then convert back to a list of strings (using the code above) on submit. See https://symfony.com/doc/5.3/form/data_transformers.html for details and examples.

Reply:
The only use of the `getStatusesByBoard` was when dealing with case filtering so it was only used for options, maybe this should have been reflected somehow, in the function name possibly.

Removed `BoardHelper` completely and used 

```
// Retrieve list of statuses by board and make them into options
                $statuses = array_filter(array_map('trim', explode(PHP_EOL, $board->getStatuses())));
                $statusOptions = [];

                foreach ($statuses as $status) {
                    $statusOptions[$status] = $status;
                }
```

to make board statuses into options.